flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/planner/plan/nodes/physical/batch/BatchPhysicalHashJoin.scala

```diff
@@ -75,7 +76,8 @@ class BatchPhysicalHashJoin(
       joinType,
       leftIsBuild,
       isBroadcast,
-      tryDistinctBuildRow)
+      tryDistinctBuildRow,
+      withHint)
```

Comment:
The name is not updated to `withJoinStrategyHint`

This also happens in a few other places.